cmd/kind/kind.go

```diff
@@ -68,14 +57,6 @@ func NewCommand() *cobra.Command {
 }
 
 func runE(flags *Flags, cmd *cobra.Command, args []string) error {
-	level := defaultLevel
```

Comment:
We should have a new flag for the klog level

Reply:
I don't think we can explicitly add our own flags for this - instead, we should be calling `klog.InitFlags` with the cobra command's flagset: https://github.com/kubernetes/klog/blob/master/klog.go#L406-L420

Reply:
Forgot about `InitFlags`. Let me add it

Reply:
We can plumb through only the flags we want. At minimum v, the others are more detabateble.